app/models/member.rb

```diff
@@ -51,6 +53,21 @@ def first_name
   def self.active
     Member.where(skip_meetings: false)
   end
+
+  ###########################################################
+  ################ Email Confirmation Part ##################
+  ###########################################################
+  def email_activate
+    self.email_confirmed = true
+    self.confirm_token = nil
+    save!(:validate => false)
```

Comment:
Why is this a `validate: false` save? I think this should be a 1-liner, like `update_attributes!(email_confirmed: true, confirm_token: nil)`.